src/pycodestyle/plugins.rs

```diff
@@ -201,3 +201,62 @@ pub fn literal_comparisons(
 
     checker.add_checks(checks.into_iter());
 }
+
+/// E713, E714
+pub fn not_tests(
+    checker: &mut Checker,
+    expr: &Expr,
+    op: &Unaryop,
+    operand: &Expr,
+    check_not_in: bool,
+    check_not_is: bool,
+) {
+    if matches!(op, Unaryop::Not) {
+        if let ExprKind::Compare {
+            left,
+            ops,
+            comparators,
+            ..
+        } = &operand.node
+        {
+            let should_fix = ops.len() == 1;
```

Comment:
Should we fix `not (x in y in z)` (= `not (x in y and y in z)`) to `x not in z or y not in z`?

Reply:
I think it's fine to omit those. They're less in the spirit of the rule.

Reply:
It looks like Flake8 doesn't even flag those as errors.

Reply:
```
> cat resources/test/fixtures/a.py
if not a in b in c:
    pass
> flake8 resources/test/fixtures/a.py --select E713
resources/test/fixtures/E713.py:1:4: E713 test for membership should be 'not in'
```

Reply:
Haha :)